waspc/headless-test/examples/todoApp/todoApp.wasp

```diff
@@ -20,6 +20,7 @@ app todoApp {
           clientRoute: PasswordResetRoute
         }
       },
+      discord: {},
```

Comment:
No need to modify the headless tests to include Discord 👍 we only included the Google one to test out the social login provider React components.

I'd remove any Discord related changes in the headless tests.